loader/status.go

```diff
@@ -63,7 +63,8 @@ func (l *Loader) PrintStatus(ctx context.Context) {
 
 		finishedSize := l.finishedDataSize.Get()
 		totalSize := l.totalDataSize.Get()
-		log.Infof("[loader] finished_bytes = %d, total_bytes = GetAllRestoringFiles%d, progress = %s", finishedSize, totalSize, percent(finishedSize, totalSize))
+		filesCount := l.db2Tables
```

Comment:
`l.db2Tables` is a map from db -> tables and table -> data files, we can't get AllRestoringFiles count directly.

We have two options to get this count:
1. count db, table, and data files via `l.db2Tables`
2. update this count during `l.prepare()`

Reply:
what do you think？ @ddatsh

Reply:
first, we just print with  l.db2Tables only?

Reply:
yep, `l.db2Tables` contains all db schema, table schema, table table files that we concern (concern here means we have filtered system schema and schema/tables in black list). Count files stored in `l.db2Tables` is enough